src/api/requests/downloadValidationReport.ts

```diff
@@ -49,7 +49,7 @@ export const downloadValidationReport = async ({
       link.href = url;
       link.setAttribute(
         'download',
-        `${lei}-${filingPeriod}-submission-${submissionId}-validation_report.csv`,
+        `${lei}_${filingPeriod}_${submissionId}_validation_report.csv`,
```

Comment:
Changed this to line up with conversations with Natalia. Will be revisited later.